### DISCUSSION TITLE
can't re-install the app after delecting package-lock.json and node module

### DISCUSSION THREAD

**Discussion by anagobabatunde** [mkosir/express-typescript-typeorm-boilerplate#83] [General]
Hello everyone,
after cloning the repo and make some change , i delete the package-lock.json and nodemodule to re-install and relaunch the app , and i get this error when i try to re-install
<img width="711" alt="Capture d’écran 2021-06-28 à 14 33 52" src="https://user-images.githubusercontent.com/41580129/123637069-de4a6200-d81d-11eb-897d-5adfd3601eaa.png">

**Comment by mkosir** [comment]
Hi @anagobabatunde, I can't reproduce your error, are you using Node v14, same as it used and tested in [docker image](https://github.com/mkosir/express-typescript-typeorm-boilerplate/blob/master/Dockerfile#L1)?

**Reply by anagobabatunde** [reply]
okay thanks , it working now ; i update to node v14 thanks,
but now i have another probleme, i  use mongo as database , can you provide us a way to use mongodb as database because when i setup it with typeorm i get an error
Best

**Reply by mkosir** [reply]
True you could switch TypeORM database provider to MongoDB and additionally make changes on other places (no migrations to execute, docker image etc.), but in that case I would rather suggest using different boilerplate to start from, since this one is tailored towards Postgres.
